src/Domain/CoordinateMaps/Frustum.hpp

```diff
@@ -271,11 +272,10 @@ class Frustum {
    * \param orientation_of_frustum The orientation of the frustum in 3D space.
    * \param with_equiangular_map Determines whether to apply a tangent function
    * mapping to the logical coordinates (true) or not (false).
-   * \param projective_scale_factor The scale factor of the projective map
-   * applied to the zeta logical coordinate.
-   * \param auto_projective_scale_factor Determines whether to automatically
-   * calculate the projective scale factor based on the frustum geometry (true)
-   * or use the provided scale factor (false).
+   * \param zeta_distribution Whether to apply a linear, logarithmic, or
```

Comment:
`radial_distribution`?

Reply:
I want to keep this as `zeta_distribution`, at least in the Frustum, because this Distribution map is applied to the zeta direction, which is not necessarily the radial direction (though they coincide in the BBH domain).

Reply:
ok